docs/graphQl/TestingVSCodeExtension/GetPatients.http

```diff
@@ -0,0 +1,109 @@
+# .SUMMARY This test flow confirms that we handle graphql requests
+# for the Patient resource.
+# The assumption for the requests and resources below:
+# 1. The FHIR version is R4
+# 2. FHIR-Server is working in the branch: feature/graphql-prototype
+# 3. appsettings.json has Security.Enabled = false
+
+@hostname = localhost:44348
+
+### Test REST client
+https://{{hostname}}/metadata
+
+
+### Post one Patient resource (using REST API),
+# to test the scenarios below:
+POST https://{{hostname}}/Patient
+content-type: application/json
+
+< .\Data\MockInformation.json
+
+
+### Get all Patients Id in the server using GraphQl by GET.
+# You should see at least one id.
+GET https://{{hostname}}/graphql?query={ patients { id } }
+Content-Type: application/json
+
+
+### Get Patients Id in the server using GraphQl by POST.
+# You should see at least one id.
+POST https://{{hostname}}/graphql
+Content-Type: application/json
+X-Request-Type: GraphQL
+
+query {
+  patients {
+    id
+  }
+}
+
+### Get Patients with different fields using GraphQl by POST REST.
+# You should see at least one patient.
+POST https://{{hostname}}/graphql
+Content-Type: application/json
+X-Request-Type: GraphQL
+
+query {
+  patients {
+    id
+    name {
+        use
+        text
+        family
+        given
+    }
+    address {
+        line
+        city
+        state
+    }
+  }
+}
+
+### Get Patient by ID with using GraphQL.
+# To modify the following request, first get all patients
+# in the server with query shown above (line 28). Then,
+# pick one ID from your response and past it to the line 83.
+
+POST https://{{hostname}}/graphql
+Content-Type: application/json
+X-Request-Type: GraphQL
+
+query($id: String) {
+    patientById(id: $id) {
+        id
+        name {
+            family
+        }
+    }
+}
+
+{
+    "id": "8dedbb6f-482f-4b40-b57e-1b4cfec01b52"
+}
+
+
+### Get 2 Patients by ID with different fields using GraphQl.
+# To modify the following request, first get all patients
+# in the server with query shown above (line 28). Then,
+# pick 2 IDs from your response and paste it into lines 107 y 108.
```

Comment:
Nit: I think the 'y' between the line numbers should be an &

Reply:
Yes, I wrote the AND in Spanish, thank you RB!

Reply:
I'm learning! I thought you had tried to type an & and just hit y instead :)